### DISCUSSION TITLE
srt-live-transmit stats question

### DISCUSSION THREAD

**Discussion by sartoricz** [Haivision/srt#2624] [Q&A] [answered]
Hello,

I wonder where the "sid" key in stats comes from, can it be pre-defined when srt-live-transmit starts?

 {"sid":296028110,"timepoint":"2023-01-23T00:24:00.173475+0100","time":198719437,"window":{"flo...

Thank you.

**Comment by ethouris** [comment, marked as answer]
"sid" is "socket ID", the socket's numerical value, the one that is returned by `srt_create_socket()` and used in the operations. This can be normally ignored if you have a single SRT connection on one end, but it may be important in case when you do SRT-to-SRT flipping or a group-connection (actually not available in srt-live-transmit, but the same is available in srt-test-live).

**Reply by sartoricz** [reply]
@ethouris  Thank you for your answer. I am using several srt-live-transmit in my lab environment, fired up by systemd and stats collected from stdout with systemd journal and this sid looks a sa good discriminator between processes.
